docs/en/developer/create-web-component/VUE.md

```diff
@@ -461,3 +564,26 @@ task cleanUp(type: Delete) {
 
 jar.finalizedBy cleanUp
 ```
+
+## C. Node.js installation
+
+### Mac OS X
+
+1. [MacPorts](https://www.macports.org/)
+2. [HomeBrew](https://brew.sh/)
+3. macOS installer (.pkg) from [Node.js website](https://nodejs.org/)
+
+#### With MacPorts
```

Comment:
```
##### With MacPorts
```

Reply:
I'm leaving it as is because #### With MacPorts is the logical lower hierarchy after ### Mac OS X.